src/wallet/UserWalletTransactions.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './UserWalletSummary.less';
+import './UserWalletTransactions.less';
+import ReceiveTransaction from './ReceiveTransaction';
+import TransferTransaction from './TransferTransaction';
+import SavingsTransaction from './SavingsTransaction';
+import PowerUpTransaction from './PowerUpTransaction';
+import ClaimReward from './ClaimReward';
+
+const UserWalletTransactions = ({ transactions, currentUsername }) => (
+  <div className="UserWalletTransactions">
+    {transactions
+      .map((transaction, index) => {
+        const key = `${transaction.trx_id}${index}`;
```

Comment:
Is it necessary to append map's `index` to key? Can there be duplicate transactions?

Reply:
@Sekhmet, yeah I have had cases where there were duplicate transaction ids :(